XROLE/My-Diary
# Ch update readme #159450893

## What does this PR do?
- Add badges for Travis and coveralls

##Description of task to be completed
- install nyc, coveralls, add .coveralls.yml file and configured the file

##How should the user test it manually
- navigate to the root directory of mydiary app to see the coveralls file.

#Any background context you want to  you want to provide?
null
